uaclient/entitlements/base.py

```diff
@@ -38,6 +38,9 @@ class UAEntitlement(metaclass=abc.ABCMeta):
     # Wheter that entitlement is in beta stage
     is_beta = False
```

Comment:
let's add the "is_beta" but only if it's enabled and it's a feature that needs --beta on the cli to enable it please.